### PR TITLE
added a donator item, fixed two others

### PR DESCRIPTION
**Pull request by Useroth**
the vox mask now looks the way it should when worn by vox
the banded uniform can now have sleeves rolled up proper

added the license to many things (not actually endorsed by nanotrasen)

### CODE REVIEW

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -1241,3 +1242,41 @@
...
+	add_fingerprint(user)
+
+/obj/item/card/fuzzy_license/attackby(obj/item/P, mob/living/user, params)
+	if(user.ckey != "fuzlet")
+		return
+
+	if(istype(P, /obj/item/pen) || istype(P, /obj/item/toy/crayon))
+		var/choice = input(user, "Select the license type", "License Type Selection") as null|anything in possible_types
+		if(!isnull(choice))
+			name = "license to [choice]"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
/obj/item/card/fuzzy_license/attackby(obj/item/used, mob/living/user, params)
	if(user.ckey != "fuzlet")
		return

	if(istype(used, /obj/item/pen) || istype(used, /obj/item/toy/crayon))
		var/choice = input(user, "Select the license type", "License Type Selection") as null|anything in possible_types
		if(!isnull(choice))
			name = "license to [choice]"
```

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 2]
```diff
@@ -1241,3 +1242,41 @@
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/glasses.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/eyes.dmi'
 	icon_state = "rose"
+
+/obj/item/card/fuzzy_license
+	name = "license to hug"
+	desc = "A very official looking license. Not actually endorsed by Nanotrassen."
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
	desc = "A very official looking license. Not actually endorsed by NanoTrasen."
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm (hunk 1 above)
Comment (Useroth) [reply modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]: to be honest it was copypaste from paper code

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm (hunk 2 above)
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
	desc = "A very official looking license. Not actually endorsed by Nanotrasen."
```

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 3]
```diff
@@ -1241,3 +1242,41 @@
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/glasses.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/eyes.dmi'
 	icon_state = "rose"
+
```
Comment (thestubborn) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
// Donation reward for Fuzlet
```

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 4]
```diff
@@ -1241,3 +1242,41 @@
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/glasses.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/eyes.dmi'
 	icon_state = "rose"
+// Donation reward for Fuzlet
+/obj/item/card/fuzzy_license
+	name = "license to hug"
+	desc = "A very official looking license. Not actually endorsed by NanoTrasen."
```
Comment (tf-4) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm:1248]:
```suggestion
	desc = "A very official looking license. Not actually endorsed by Nanotrasen."
```

**Review by tf-4 — CHANGES_REQUESTED** [review]
You resolved my review, but you did not resolve my review... :( Golden was wrong, "Nanotrasen" does not have a capital "t".

**Pull request merged**